torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1340,13 +1354,14 @@ def create_unbacked_symint(self):
     # This is guaranteed to return a symbol or its negation is a sympy.Symbol,
     # but there may be a replacement that allows it to be immediately
     # simplified
-    def create_symbol(self, val: int, source: Source, dyn=False) -> "sympy.Expr":
+    def create_symbol(self, val: int, source: Source, dim_state=DIM_DYNAMISM_STATE) -> "sympy.Expr":
```

Comment:
As is, you need to assert dim_state is not static. But I still think it is profitable to make this handle static too.